tensorflow_blade/src/BUILD

```diff
@@ -2,11 +2,72 @@ load("@bazel_skylib//lib:selects.bzl", "selects")
 load("@local_config_mkl//:build_defs.bzl", "if_mkl_enabled")
 load("@pybind11_bazel//:build_defs.bzl", "pybind_extension")
 load("@local_config_tf//:build_defs.bzl", "tf_copts", "tf_version")
-load("//:build_defs.bzl", "device_name")
-load("@local_config_blade_helper//:build_defs.bzl", "python_bin_path")
+load("@local_config_blade_helper//:build_defs.bzl", "if_tensorrt_enabled", "if_internal")
 
 package(default_visibility = ["//visibility:public"])
 
+##############################################tf_blade ops################################################
+# Nested select is not supported by bazel, use an intermediat target to workaround.
+cc_library(
+    name = "tensorrt_ops",
+    deps = if_tensorrt_enabled([
+        "//src/custom_ops/trt_engine_op",
+    ]),
+)
+cc_library(
+    name = "internal_gpu_ops",
+    deps = if_internal([
+        "@org_aliyun_blade//tf_blade:tf_blade_gpu_ops_internal",
+    ]),
+)
+cc_library(
+    name = "internal_cpu_ops",
+    deps = if_internal([
+        "@org_aliyun_blade//tf_blade:tf_blade_cpu_ops_internal",
+    ]),
+)
+
+cc_library(
+    name = "tf_blade_ops",
+    deps = select({
+        "//:gpu": [
+            "//src/custom_ops/non_max_suppression",
+            "//src/custom_ops/rnn_ops",
+            ":tensorrt_ops",
+            ":internal_gpu_ops",
+        ],
+        "//:cpu": [
+            ":internal_cpu_ops",
+        ],
+        "//conditions:default": [],
+    }),
+)
+
+cc_binary(
+    name = "tf_blade",
+    copts = tf_copts(),
+    linkopts = [
+        "-ldl",
+        "-lm",
+        # Tricky... to workaroudn undefined symbols _ZdlPvm. Bazel has already add -lstdc++
+        # to linkopts before object files, but it doesn't work. By adding -lstdc++ to end of
+        # linkopts, it works, it works. I've no idea why.
+        "-lstdc++",
```

Comment:
This could be removed, i think.

Reply:
Done, it works okay without this "-lstdc++"